src/constants/SearchOptions.js

```diff
@@ -0,0 +1,20 @@
+export const SERVICES = {
+  title: "Kategoria",
+  items: ["adopcja", "hotel", "usługi"],
+};
+
+export const DATALIST = {
```

Comment:
If other lists are called by what they are, then why the data list options are called `DATALIST` and not `SERVICES`? The actually `SERVICES` should be called then `CATEGORIES`.